themes/digital.gov/layouts/partials/core/featured-resource.html

```diff
@@ -1,3 +1,24 @@
+{{/* Featured Resource Component
+
+  @params {string} resourcePath - a relative hugo path to lookup a page to feature, use a forward slash before the path and no trailing slash
+  @params {string} kicker - text that is displayed above the main content
+
+  @example shortcode
+  {{< featured-resource resourcePath = "/guides/hcd/discovery-concepts"
+>}}
+{{< featured-resource resourcePath = "/resources/an-introduction-to-plain-language" >}}
+
+@example shortcode for communities, use the filepath in the second level for
+resourcePath to look up correctly
+{{< featured-resource resourcePath = "/communities/communicators" kicker = "Join the Community" >}}
+{{< featured-resource resourcePath = "/communities/multilingual" kicker = "Join the Community" >}}
+{{< featured-resource resourcePath = "/communities/plain-language-community-of-practice" kicker = "Join the Community" >}}
+{{< featured-resource resourcePath = "/communities/social-media" kicker = "Join the Community" >}}
+{{< featured-resource resourcePath = "/communities/user-experience" kicker = "Join the Community" >}}
+{{< featured-resource resourcePath = "/communities/web-analytics-and-optimization" kicker = "Join the Community" >}}
+{{< featured-resource resourcePath = "/communities/web-managers-forum" kicker = "Join the Community" >}}
+
```

Comment:
We can simplify this comment and remove the duplicate examples for clarity.
>**Note**
>Future work; Hopeful we can create a featured community partial that's an extension of this.
>It would take a kicker and a the community name, instead of the full path. We'd set `Join the {{ Community Title }} Community` as a default kicker.

```suggestion
  @example shortcode
  {{< featured-resource resourcePath = "/guides/hcd/discovery-concepts"
>}}

@example Community shortcodes

{{< featured-resource resourcePath = "/communities/communicators" kicker = "Join the Community" >}}

Community resource paths:
- communicators
- multilingual
- plain-language-community-of-practice
- social-media
- user-experience
- web-analytics-and-optimization
- web-managers-forum

```

Reply:
Like this much better, thanks for the documentation editing! I have updated an existing trello card to for creating a new `featured-community` component and short code.